frontend/src/store/modules/models.js

```diff
@@ -35,9 +35,7 @@ const actions = {
   },
   /* eslint-disable no-shadow */
```

Comment:
Minor detail but I think this comment could be removed now as well after removing the state.

Reply:
From what I could see, that comment is needed for the `getters`.

Reply:
Ah that's right, my bad.